examples/proc_GDS_pulse_capture.py

```diff
@@ -115,50 +117,18 @@ def indiv_plots(d, thislabel, thiscolor):
         beta["t_pulse"] *= amplitude
         beta.rename("t_pulse", "$A t_{pulse}$")
         fl.plot(beta, "o", color=thiscolor, label=thislabel)
-        t_v_beta = beta.shape.alloc(dtype = np.float64).rename("$A t_{pulse}$", "beta")
+        beta.rename("$A t_{pulse}$", "t_pulse")
+        beta["t_pulse"] /= amplitude
+        t_v_beta = beta.shape.alloc(dtype = np.float64).rename("t_pulse", "beta")
         t_v_beta.setaxis("beta", beta.data)
-        t_v_beta.data[:] = beta.getaxis("$A t_{pulse}$")
-        t_v_beta.sort("beta")
-        codehasbeenreviewed = True
-        if codehasbeenreviewed:
-            if amplitude > 1:
-                linear_regime = (7,None)
-            else:
-                linear_regime = (2,None)
-            c_nonlinear = beta.polyfit("$A t_{pulse}$", order = 10)
-            c_linear = beta["$A t_{pulse}$":linear_regime].polyfit("$A t_{pulse}$", order = 1)
-            print(c_nonlinear)
-            print(c_linear)
-            fit_beta_v_t = np.polyval(c_nonlinear[::-1], beta.getaxis("$A t_{pulse}$"))
-            fit = psd.nddata(fit_beta_v_t, "$A t_{pulse}$").setaxis(
-                "$A t_{pulse}$", beta.getaxis("$A t_{pulse}$")
-            )
-            fl.plot(fit, color=thiscolor, ls=":", alpha=0.5)
-            psd.gridandtick(plt.gca())
-            plt.ylabel(r"measured $\beta$ / $\mathrm{\mu s \sqrt{W}}$")
-            plt.xlabel(r"Amplitude*$t_{pulse}$ / $\mu$s")
-            # }}}
-            # {{{ t vs beta
-            fl.next(r"Amplitude*$t_{pulse}$ vs Measured $\beta$")
-            fl.plot(t_v_beta, "o", color=thiscolor, label=thislabel)
-            if amplitude > 1:
```

Comment:
do not resuscitate this stuff that I'm deleting.  all the info you need is in the code that I added and in my PR comments.  This stuff is going in a bad direction.